controllers/account.controller.js

```diff
@@ -1,53 +1,127 @@
 const Account = require("../schema/account.schema");
+const Balance = require("../schema/balance.schema");
 const { ObjectId } = require("mongoose").Types;
 const { JSONResponse } = require("../utilities/jsonResponse");
+const { randomNumberGenerator } = require("../utilities/randomNumbersGenerator");
+
 class AccountController{
     
     static createAccount = async (req, res, next)=>{
+        let accountBal = "";
         try{
             let data = req.body;
+            data.id_type = data.id_type.toUpperCase();
             if(Object.keys(data).length == 0)throw new Error("No data was passed to create account");
-            const account = await new Account.create(data);
-            JSONResponse.success(res,"Account was successfully created", account, 201);
+            let account = new Account(data);
+            accountBal = await this.generateBalanceId(data);
```

Comment:
when i create an account the account balance is the accountId

Reply:
when this is fixed i can test the balance in transactions

Reply:
it is the account balance id...they are linked to the account balance in the table..if you want me to populate it then thats fine too.

Reply:
projected the account balance and _id to be returned and assigned to the account balance